README.md

```diff
@@ -355,7 +355,11 @@ nginx_http_template:
     template_file: http/default.conf.j2
     conf_file_name: default.conf
     conf_file_location: /etc/nginx/conf.d/
-    port: 8081
+    listen:
+      port: 8081
+      opts: [] # Listen opts like http2 which will be added (ssl is automatically added if you specify 'ssl:').
```

Comment:
There's a slight misalignment in this line

Reply:
Hey @alessfg ,
which kind of misalignment? I don't see it.

Reply:
L359 and L360 -- the port and opts parameters are not equally indented :)

Reply:
I checked at here on GitHub and also on my local files. both are equally intended.
I copied this from my local files:  
```
    listen:
      port: 8081
      opts: [] # Listen opts like http2 which will be added (ssl is automatically added if you specify 'ssl:').
      ipv4: true
      ipv6: false
```

I'm sorry. Can you show me how do you think it should look correct?

If you mean that `port` and `opts` are intended under `listen` this is correct, is the mentioned break of the current behavior how to set the port.

Reply:
Oh yeah you are absolutely right! Sorry for the confusion 😕 GitHub was (still is) displaying the listen block in the README with `ports` and `opts` having different indentations 🤷‍♂

Reply:
OK. I thought I had some problems with my eyes 😅 

*Btw. for me It looks like that they have the same indentation.
Quiet weird.